### PR TITLE
Add redirect uri in request

### PR DESCRIPTION
**Pull request by sameerag**
Add Desc later @sameerag

### CODE REVIEW

File: lib/msal-core/test/UserAgentApplication.spec.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ import { ServerRequestParameters } from "../src/ServerRequestParameters";
 import { TEST_URIS, TEST_DATA_CLIENT_INFO, TEST_HASHES, TEST_TOKENS, TEST_CONFIG, TEST_TOKEN_LIFETIMES } from "./TestConstants";
 import { IdToken } from "../src/IdToken";
 import { TimeUtils } from "../src/utils/TimeUtils";
+import { request } from "http";
```
Comment (DarylThayil) [inline lib/msal-core/test/UserAgentApplication.spec.ts]: did you mean to import this ?

Comment (sameerag) [reply lib/msal-core/test/UserAgentApplication.spec.ts]: oops, thanks. I think it is vscode being "smart"

Comment (DarylThayil) [reply lib/msal-core/test/UserAgentApplication.spec.ts]: ok approved when its removed, feel free to merge

Comment (sameerag) [reply lib/msal-core/test/UserAgentApplication.spec.ts]: Done. Please check.

**Pull request merged**